gms/gossiper.cc

```diff
@@ -2026,11 +2026,9 @@ future<> gossiper::start_gossiping(gms::generation_type generation_nbr, applicat
         local_state.add_application_state(entry.first, entry.second);
     }
 
-    auto generation = local_state.get_heart_beat_state().get_generation();
+    co_await replicate(get_broadcast_address(), local_state, permit.id());
 
-    co_await replicate(get_broadcast_address(), std::move(local_state), permit.id());
-
-    logger.trace("gossip started with generation {}", generation);
```

Comment:
Shouldn't we keep the generation (at least in trace)? @asias

Reply:
It's printed below as part of `local_state`.
Previously `local_state` was moved to `replicate` so we needed to copy `generation` first before moving it, but this change keeps it so we can print the whole endpoint_state, not just the generation.

Reply:
ok